helmfile/helmfile.yaml

```diff
@@ -0,0 +1,21 @@
+repositories:
+
+  - name: cloudnativedevops
+    url: https://raw.githubusercontent.com/cloudnativedevops/cloudnatived/master/hello-helm/
+
+releases:
+
+  - name: production-demo
```

Comment:
This may be too much of a nitpick, but would it be better to have the name (and namespace) the other way round, as `demo-production`? That way, the different environments for the same app all sort together alphabetically.